test/metabase/api/action_test.clj

```diff
@@ -238,6 +230,45 @@
           (is (= result-count (count (mt/rows (qp/process-query query-that-returns-more-than-one))))
               "The result-count after a rollback must remain the same!"))))))
 
+(deftest row-delete-unparseable-values-test
+  (testing "POST /api/action/row/delete"
+    (testing "should return error message if value cannot be parsed correctly for Field in question"
+      (mt/test-drivers (mt/normal-drivers-with-feature :actions)
+        (actions.test-util/with-actions-test-data-and-actions-enabled
+          (is (schema= {:message  #"Error filtering against :type/(Big)?Integer Field: unable to parse String \"one\" to a :type/(Big)?Integer"
+                        s/Keyword s/Any}
+                       ;; TODO -- this really should be returning a 400 but we need to rework the code in
+                       ;; [[metabase.driver.sql-jdbc.actions]] a little to have that happen without changing other stuff
+                       ;; that SHOULD be returning a 404
+                       (mt/user-http-request :crowberto :post 404
+                                             "action/row/delete"
+                                             (mt/mbql-query categories {:filter [:= $id "one"]})))
+              "Delete should return the right shape")
+          (testing "no rows should have been deleted"
+            (is (= 75
+                   (categories-row-count)))))))))
```

Comment:
This test is actually not related to this bug or the fix but I wrote it while trying to figure out what the cause of the bug was. We didn't have any other tests like it so it made sense to keep it